http/influxdb/bucket.go

```diff
@@ -1,80 +0,0 @@
-package influxdb
-
-import (
-	"context"
-	"fmt"
-	"time"
-
-	platform2 "github.com/influxdata/influxdb/v2/kit/platform"
-
-	platform "github.com/influxdata/influxdb/v2"
-	"github.com/influxdata/influxdb/v2/kit/tracing"
-)
-
-// BucketService connects to Influx via HTTP using tokens to manage buckets
-type BucketService struct {
```

Comment:
There sure are a lot of bucket services in this application! Fortunately, this one wasn't used anywhere else.